core/src/main/java/com/radixdlt/ledger/StateComputerLedger.java

```diff
@@ -280,12 +294,20 @@ public EventProcessor<BFTCommittedUpdate> bftCommittedUpdateEventProcessor() {
       var proof = committedUpdate.vertexStoreState().getRootHeader();
       var transactionsWithProof = CommittedTransactionsWithProof.create(transactions, proof);
 
-      this.commit(transactionsWithProof, committedUpdate.vertexStoreState());
+      this.commitWithTimer(transactionsWithProof, committedUpdate.vertexStoreState());
     };
   }
 
   public EventProcessor<CommittedTransactionsWithProof> syncEventProcessor() {
-    return p -> this.commit(p, null);
+    return p -> this.commitWithTimer(p, null);
+  }
+
+  private void commitWithTimer(
```

Comment:
The `prepareWithTimer()` and `commitWithTimer()` methods start to look like a recurring pattern.
What do you think about adding a convenience `.measure(Runnable/Callable<T>)` method to our `Timer` class, instead?
(the impl will simply forward to the existing underlying `Summary.time(Runnable/Callable<T>)`)